src/agent/state.ts

```diff
@@ -32,7 +32,7 @@ import {DebugAgentConfig} from './config';
 // TODO: Determine if `ScopeType` should be named `scopeType`.
 // tslint:disable-next-line:variable-name
 const ScopeType = vm.runInDebugContext('ScopeType');
-const assert = debugAssert(process.env.CLOUD_DEBUG_ASSERTIONS);
+const assert = debugAssert(!!process.env.CLOUD_DEBUG_ASSERTIONS);
```

Comment:
Would the tests break without the addition of `!!` here?

Reply:
As debugAssert takes boolean and process.env.CLOUD_DEBUG_ASSERTIONS is a null or string. Converting to boolean value is needed.

Reply:
@DominicKramer `@types/node` changed the type of `process.env` from `any` to `{ [p:string]: string|undefined }` from v7 to v8.